python/lsst/pipe/tasks/propagateSourceFlags.py

```diff
@@ -189,6 +189,10 @@ def run(self, coadd_object_cat, ccd_inputs,
                     for row in ccd_inputs[ccd_inputs["visit"] == visit]:
                         detector = row["ccd"]
                         wcs = row.getWcs()
+                        if wcs is None:
+                            self.log.info("No WCS for visit %d detector %d, so can't match sources to "
+                                          "propagate flags.  Skipping...", visit, detector)
+                            continue
```

Comment:
This is fine as a belt-and-suspenders, but I don't know how a None wcs can possibly make it into the coadd input list.